pkg/protocompat/time.go

```diff
@@ -16,6 +16,20 @@ var (
 	TimestampType = reflect.TypeOf(gogoTimestamp.Timestamp{})
 )
 
+// Timestamp represents a point in time independent of any time zone or local calendar, encoded
```

Comment:
Did you wrote it on your own or it's copied from somewhere?

Reply:
I copied it from the documentation of the underlying type.
I think the more comprehensive version is on the well known protobuf types: https://pkg.go.dev/google.golang.org/protobuf/types/known/timestamppb#Timestamp